portfolio/src/main/webapp/script.js

```diff
@@ -97,23 +133,24 @@ function createMap() {
 }
 
 async function getData(maxComments) {
-    if (!maxComments) maxComments = defaultMaxComments;
-    try {
-        const ret = await fetch('/data?maxComments=' + maxComments);
-        const comments = await ret.json();
-        const commentDiv = document.querySelector('#comments');
-        commentDiv.innerHTML = "";
-        comments.forEach((comment, i) =>
-            commentDiv.innerHTML += `<p>Comment ${i + 1} is ${comment}</p>`);
-    } catch (e) {
-        alert(`Hey!! There was an error: ${e?.message || e}`);
-    }
+  if (!maxComments) maxComments = defaultMaxComments;
+  try {
+      const ret = await fetch('/data?maxComments=' + maxComments);
+      const comments = await ret.json();
+      const commentDiv = document.querySelector('#comments');
+      commentDiv.innerHTML = "";
+      comments.forEach((comment, i) =>
+          commentDiv.innerHTML += `<p>Comment ${i + 1} is ${comment}</p>`);
+  } catch (e) {
+      alert(`Hey!! There was an error: ${e?.message || e}`);
+  }
 }
 
 function init() {
```

Comment:
nit: I would put the init function right below constructor. It's pretty common to have these as the first two for easy readability / understanding of what the page is doing